### ISSUE TITLE
Pomalé načítání obsahu databáze

### ISSUE THREAD

**Issue by Grejdy** [bosancz/bosan.cz#488]
Při otevření databáze v interní sekci trvá, celkem dlouho, než se načte její obsah (zatímco samotná stránka je už plně načtená) - po tuto dobu to vypadá, že je databáze prázdná a stránka nereaguje. Na počítači to takový problém není (tam se doba načítání pohybuje okolo jedné vteřiny - aspoň u mne), ale na mobilu si občas člověk říká, jestli to vůbec funguje - zkoušel jsem dva prohlížeče: Vivaldi (jádro chromium) - 8 vteřin a Firefox - 13 vteřin.

Myslím, že by bylo vhodné optimalizovat načítání pro méně výkonná zařízení. Napadá mne, že by mohlo pomoci třeba zmenšit množství dat, co se musí načíst najednou při startu stránky (další by se mohly načítat třeba při scrollování), nebo něco takového.

**Comment by SmallhillCZ** [comment]
Jo ona se totiž načítá celá. Ale těch dat je pomálu, rozhodně míň, než průměrný obrázek. Možná se jen zobrazuje pomalu, to stahování dat by problém být neměl.

**Comment by Grejdy** [comment]
No, každopádně vždycky, když otvírám na telefonu databázi, tak to trvá půl věku a myslím, že by bylo příjemné, kdyby se to změnilo. To s těma datama byl jen nápad, čím by to mohlo být (resp. nenapadá mě jiný důvod, proč by to tomu mělo tak trvat).

Každopádně asi to bude souviset s tím, kolik se toho načítá, protože když tam pak přidávám nějaké filtry (konkrétně to zkouším na oddílu) tak ta prodleva závisí na tom, kolik toho tam nechám načítat - celá databáze aktivních členů: 12 vteřin, databáze členů třeba čtvrtého a šestého oddílu: 2 vteřiny, databáze čtyř oddílů (třeba 4, 6, 7, 8): 5vteřin, šest oddílů: 8 vteřin…

A pro jistotu jsem zkoušel i jiné zařízení (aby to nebylo náhodou mým telefonem) a vypadá to, že to tak je na většině telefonů.

**Comment by Grejdy** [comment]
Jo a ještě mám k databázi jednu vítku, kterou nevím, jestli dávat do samostatné issue, když se stejně asi někdo bude hrabat v nastavení databáze kvůli tomuhle.
Všiml jsem si, že v mobilním zobrazení databáze nenechává dost místa pro poslední položku, takže když člověk doscrolluje úplně na konec stránky, jeden údaj zasahuje mimo obrazovku.
![image](https://user-images.githubusercontent.com/82664677/170263301-f32398a1-76ae-4fd2-811d-83f608bac434.png)
